cmd/ocm/list/version/cmd.go

```diff
@@ -35,6 +39,17 @@ var Cmd = &cobra.Command{
 	RunE: run,
 }
 
+func init() {
+	fs := Cmd.Flags()
+	fs.BoolVarP(
+		&args.defaultVersion,
+		"default",
+		"d",
+		false,
+		"Show the default version",
```

Comment:
Consider something like "Show the default version instead of listing all versions".

Reply:
or "Show only the default version".

Reply:
LGTM otherwise +1